dvc/repo/ls.py

```diff
@@ -82,3 +113,22 @@ def _get_target_path_info(repo, target=None):
     if not target:
         return PathInfo(repo.root_dir)
     return PathInfo(repo.root_dir, target)
+
+
+def get_fs_node(path_info, isout=False):
```

Comment:
```suggestion
def _get_fs_node(path_info, isout=False):
```
since it is really a private method.